views/partials/docs/moons/getByMonth.pug

```diff
@@ -0,0 +1,22 @@
+div(class="section")
+  div(class="lable") Method
+  div(class="value delete") get
+div(class="section")
+  div(class="lable") Route
+  div(class="value") http:localhost:2319/moons/month/{monthId}
```

Comment:
This route doesn't work

![image](https://user-images.githubusercontent.com/5775716/84069598-c5fc7080-a998-11ea-8138-c5d8fd340844.png)